jetty-websocket/jetty-websocket-server/src/main/java/org/eclipse/jetty/websocket/server/WebSocketServlet.java

```diff
@@ -16,7 +16,7 @@
 //  ========================================================================
 //
 
-package org.eclipse.jetty.websocket.servlet;
+package org.eclipse.jetty.websocket.server;
```

Comment:
Rather than just move the Servlet to server, it would be better to leave a version of it behind in servlet... perhaps even basing there server version on the servlet version.

So is it possible to have:
+ ws.servlet version that has API independent behaviour... perhaps with some deprecated methods that support the old jetty API (but maybe not as this is 10)
+ ws.server version that extends the ws.servlet version to add the Jetty API specific behaviour